kafka_consumer/hatch.toml

```diff
@@ -3,27 +3,15 @@
 
 [[envs.default.matrix]]
 python = ["3.8"]
-version = ["0.11", "1.1", "2.3", "latest"]
-offset-storage = ["kafka", "zk"]
+version = ["1.1", "2.3", "latest"]
 
 [envs.default.overrides]
-matrix.version.e2e-env = { value = true, if = ["0.9", "latest"] }
+matrix.version.e2e-env = { value = true, if = ["latest"] }
```

Comment:
we were not testing 0.9, we should look into that, does that mean we're not running any e2e tests?